LICENSE

```diff
@@ -1,3 +1,10 @@
+The source code is licenced under Appache Licence Version 2.0 (see below).
```

Comment:
This is a little odd - I've never seen a LICENSE file that includes anything but the license text itself. I think Credits.md is a fairly reasonable location for the attribution notices, though they could be moved to a separate file (e.g. Attribution.md / ThirdPartyNotices.md).